scientific_library/tvb/simulator/simulator.py

```diff
@@ -432,9 +432,9 @@ def _apply_spike_stimulus(self, step):
             setattr(self.model, target_parameter, self._spike_stimulus_fun(target_parameter, step-1))
 
     def update_non_state_variables(self, state, node_coupling, local_coupling):
-        if self.model._update_non_state_variables:
-            state = self.model.update_non_state_variables(state, node_coupling, local_coupling,
-                                                          use_numba=self.use_numba)
+        state = self.model.update_non_state_variables(state, node_coupling, local_coupling,
```

Comment:
Did these changes in order to remove the has attr checking from the model configure method.